scripts/scil_screenshot_bundle.py

```diff
@@ -36,8 +41,8 @@ def _build_args_parser():
                    help='Path of the input bundle.')
     p.add_argument('in_anat',
                    help='Path of the reference file (.nii or nii.gz).')
-    p.add_argument('target_template',
-                   help='Path to the target MNI152template for registration. \n'
+    p.add_argument('--target_template',
+                   help='Path to the target MNI152 template for registration. \n'
                         'If in_anat has a skull, select a MNI152 template \n'
                         'with a skull and vice-versa.')
```

Comment:
What happens if people are not in MNI152 and forget to add the --target_template? Maybe when people don't add this option we should output some info like "we consider your data is already aligned in MNI."